daily-challenges/2024-12/14/problem.md

```
@@ -0,0 +1,66 @@
+# Typescript challenge: XẾP ĐỒ VÀO TỦ
+
+## Giới thiệu
+Việc sắp xếp đồ đạc vào tủ luôn là một thách thức! Bạn cần tối ưu không gian để có thể chứa được nhiều đồ nhất có thể. Hãy viết chương trình giúp sắp xếp đồ vào tủ một cách thông minh.
+
+## Yêu cầu
+- Viết hàm `xepTu(doVat, kichThuocTu)`
+- Input:
+  + doVat: mảng các object chứa thông tin đồ vật:
```

Markdownlint (0.37.0)

Expected: dash; Actual: plus
Unordered list style

(MD004, ul-style)